angular/projects/admin/src/app/admin/document/document-list/document-list.component.ts

```diff
@@ -14,6 +14,7 @@ import { tap } from 'rxjs/operators';
 })
 export class DocumentListComponent implements OnInit, OnDestroy, AfterViewInit {
   total_count: number;
+  lastPageIndex: Number = 0;
```

Comment:
Use the type `number` with all small letters (https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).

![image](https://user-images.githubusercontent.com/2051607/57908919-e3edc380-78b3-11e9-8a4f-9af0e468b939.png)

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/193/commits/e4fd812a1d7a0d05d2b802e879fae0544502fb07